app/views/render_async/_request_jquery.js.erb

```diff
@@ -112,7 +111,9 @@ if (window.jQuery) {
       });
     }
 
-    _runAfterDocumentLoaded(_setUpToggle)
+    //_runAfterDocumentLoaded(_setUpToggle)
+    // RUN IT NOW DP
+    _setUpToggle();
```

Comment:
Can you remove this line, `_setUpToggle()` is supposed to happen after the page loads, so that we don't miss any elements that are potentially loading. This was a fix for https://github.com/renderedtext/render_async/issues/90

Reply:
Regarding #90, is "lazy load" implemented yet?  Running after the document is loaded is causing issues where I want to trigger the toggle manually on page load.  Basically I use the same partial on two views.  On one view, I want the async partial to render on user command, on the other, I want the same async partial to render automatically.  This is easily done by triggering a click on the selector:  $(selector).click();  The delayed _setupToggle call does not allow us to perform the click() in the normal view/partial rendering. 

In 2.1.14, I was not experiencing the issue as described in #90.

Reply:
> Regarding #90, is "lazy load" implemented yet?

No, lazy load is not implemented yet.

> This is easily done by triggering a click on the selector: $(selector).click(); The delayed _setupToggle call does not allow us to perform the click() in the normal view/partial rendering.

Have you tried what I proposed in the comment for #108 https://github.com/renderedtext/render_async/issues/108#issuecomment-607055247?